tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
`tsconfig.json` lets you define the rules you want for your TypeScript setup
can leave this config and update if you ever need to